### ISSUE TITLE
Dynamic Prompts hasn't worked for me for a month

### ISSUE THREAD

**Issue by Xenodimensional** [adieyal/sd-dynamic-prompts#725]
Hi, DP has consistently failed to load for at least amonth and no amount of removing/repairing updating has done anything. I must have generated 20k images using it and I would like to get it working again because not having it has seriously halted my workflow and made it impossible to load all my work. PLEASE HELP. This is the error:

Traceback (most recent call last):
  File "C:\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1872, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-dynamicprompts\__init__.py", line 1, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-dynamicprompts\nodes\__init__.py", line 4, in <module>
    from .magicprompt import DPMagicPrompt
  File "C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-dynamicprompts\nodes\magicprompt.py", line 4, in <module>
    from dynamicprompts.generators.magicprompt import MagicPromptGenerator
  File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\dynamicprompts\generators\magicprompt.py", line 13, in <module>
    from transformers import (
  File "<frozen importlib._bootstrap>", line 1229, in _handle_fromlist
  File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\transformers\utils\import_utils.py", line 1272, in __getattr__
    module = self._get_module(self._class_to_module[name])
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\transformers\utils\import_utils.py", line 1284, in _get_module
    raise RuntimeError(
RuntimeError: Failed to import transformers.pipelines because of the following error (look up to see its traceback):
DLL load failed while importing _pywrap_tf2: A dynamic link library (DLL) initialization routine failed.

Cannot import C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-dynamicprompts module for custom nodes: Failed to import transformers.pipelines because of the following error (look up to see its traceback):
DLL load failed while importing _pywrap_tf2: A dynamic link library (DLL) initialization routine failed.

**Comment by akx** [comment]
This is the repository for the WebUI extension for Dynamic Prompts. The Comfy node's repository is [here](https://github.com/adieyal/comfyui-dynamicprompts).

I would recommend googling for `DLL load failed while importing _pywrap_tf2` – that's the salient part of your traceback.

**Comment by Xenodimensional** [comment]
> DLL load failed while importing _pywrap_tf2

Sorry, I just realised that. I'm not a developer or coder. I have virtually no ability to fix this. I installed the addon in good faith that it would remain functional and it has actually negatively impacted my workflow. I'm not going to mess with any files in my Comfy ecosystem, that sounds like a recipe to royally eff things up. I'm just going to have to live with the fact that the the bulk of my  time spent on the last few months of work has probably been wasted.
